cmd/heimdalld/main.go

```diff
@@ -181,16 +168,9 @@ func InitCmd(ctx *server.Context, cdc *codec.Codec) *cobra.Command {
 			}
 
 			// Heimdall config file
-			heimdallConf := getDefaultHeimdallConfig()
-			heimdallConfBytes, err := json.MarshalIndent(heimdallConf, "", "  ")
-			if err != nil {
-				return err
-			}
-
-			if err := common.WriteFileAtomic(filepath.Join(config.RootDir, "config/heimdall-config.json"), heimdallConfBytes, 0600); err != nil {
-				fmt.Println("Error writing heimdall-config", err)
-				return err
-			}
+			heimdallConf := helper.GetDefaultHeimdallConfig()
+			fmt.Println("pringitn heimdall-config", heimdallConf.MaxCheckpointLength)
```

Comment:
spell check in log. 
Also can we use logger?

Reply:
Sorry, wasn't supposed to push this!